### ISSUE TITLE
Standardize kubeconfig management

### ISSUE THREAD

**Issue by jonerer** [ssup2/kpexec#9]
Hello!

Thanks for a great project, it's very useful!

When trying to connect to my cluster I had some difficulties though. Mainly in dealing with cluster settings. It would be really nice if kpexec could manage kubernetes config in the same way as kubectl to reduce friction. Kubernetes reads the env variable KUBECONFIG to let you load a set of config files, like ``` export KUBECONFIG=~/kube/config:~/.kube/cluster.d/us.us-east-1.eksctl.io.conf```. But in kpexec this is expected to always be called `~/kube/config` unless otherwise specified by argument:

```cmd.Flags().StringVar(&options.kubeconfig, "kubeconfig", filepath.Join(homedir.HomeDir(), ".kube", "config"), "Absolute path to the kubeconfig file")```

It would be really great if kpexec could load in whatever is in the KUBECONFIG env var, or just drop any config management code and let kubectl do it in whatever way it wants by loading conf with `kubectl config view`.

TL;DR
Actual:
```
➜  export KUBECONFIG=~/.kube/config:~/.kube/cluster.d/us.us-east-1.eksctl.io.conf
➜  kubectl get pod -n kafka
NAME                                               READY   STATUS    RESTARTS   AGE
production-kafka-exporter-86856fd478-cm67l         1/1     Running   141        17h
➜  kpexec -it -T -n kafka production-kafka-exporter-86856fd478-cm67l -- bash
Failed to run kpexec err : failed to set kubeconfig : invalid configuration: [context was not found for specified context: miniadmin@us.us-east-1.eksctl.io, cluster has no server defined]
```

Expected: should work

Thanks

**Comment by ssup2** [comment]
@jonerer Thanks for the nice proposal. I'll fix it. Thanks.

**Comment by jonerer** [comment]
FYI this can be done like so:
```
	k := clientcmd.NewDefaultClientConfigLoadingRules()
	config, err := k.Load()
```

**Comment by jonerer** [comment]
FYI I just had to do this on another project and found it it's a bit tricky/badly documented. But you can do it like this:

```
	k := clientcmd.NewDefaultClientConfigLoadingRules()
	cfg, err := k.Load()
	if err != nil {
		panic(err)
	}
	clientConfig := clientcmd.NewDefaultClientConfig(*cfg, &clientcmd.ConfigOverrides{})
	restclientConfig, err := clientConfig.ClientConfig()
	if err != nil {
		panic(err)
	}

	// create the clientset
	clientset, err := kubernetes.NewForConfig(restclientConfig)
	if err != nil {
		panic(err.Error())
	}
```

**Comment by ssup2** [comment]
@jonerer Thanks. I fix this issue and release kpexec. please try it again with v0.3.5 version :)
